test/DssVestTopUp.test.ts

```diff
@@ -156,11 +157,10 @@ describe("DssVestTopUp", function () {
       });
 
       it("should vest accrued tokens", async function () {
-        const unpaid = await dssVest.unpaid(vestId);
         await topUp.refundUpkeep();
         const vested = await token.balanceOf(topUp.address);
 
-        expect(unpaid).to.eq(vested);
+        expect(vested).to.eq(maxDepositAmt);
```

Comment:
What's the rationale behind this change?

Reply:
Before these changes, there were 10 000 tokens left in the contract but now that the surplus is transferred we are only left with maxDepositAmt in the topUp contract.